draft-ietf-mls-protocol.md

```diff
@@ -1353,13 +1353,16 @@ proceeds as shown in the following diagram:
                    init_secret_[n-1]
                          |
                          V
-    commit_secret -> KDF.Extract = joiner_secret
+    commit_secret -> KDF.Extract
                          |
                          V
                    DeriveSecret(., "member")
                          |
                          V
-psk_secret (or 0) -> KDF.Extract = member_secret
+                    joiner_secret
```

Comment:
Nit: Either this should be `member` or the label above should be `joiner`

Reply:
Changed it to `joiner` as I think calling this the _"joiner_secret"_ makes more sense, but would be happy to do the change the other way as well.